core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -2407,6 +2417,32 @@ class EMConfig {
     @Cfg("Apply a security testing phase after functional test cases have been generated.")
     var security = true
 
+    @Experimental
+    @Cfg("To apply SSRF detection as part of security testing.")
+    var ssrf = false
+
+    enum class VulnerableInputClassificationStrategy {
+        /**
+         * Uses the manual methods to select the vulnerable inputs.
+         */
+        MANUAL,
+
+        /**
+         * Use LLMs to select potential vulnerable inputs.
+         */
+        LLM,
+    }
+
+    @Experimental
+    @Cfg("HTTP callback verifier port.")
```

Comment:
add some more info. eg `A server could be started locally to verify security attacks such as SSRF.`